client/blocks/reader-full-post/wp-iframe-resize.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * This function is inspired by `wp-includes/js/wp-embed.js` of WP.org.
+ * It actually waits for a message from within the iFrame.
+ * The message will contain the actual height of the iFrame.
+ *
+ * @param {Element} contentWrapper The content wrapper element.
+ * @param {boolean} removeListener When true, remove the listener instead of adding it.
+ */
+const WPiFrameResize = ( contentWrapper, removeListener ) => {
+	const receiveEmbedMessage = function ( e ) {
+		const data = e.data;
+
+		if ( ! data ) {
+			return;
+		}
+
+		if ( ! ( data.secret || data.message || data.value ) ) {
+			return;
+		}
+
+		if ( /[^a-zA-Z0-9]/.test( data.secret ) ) {
+			return;
+		}
+
+		const iframes = contentWrapper.querySelectorAll( 'iframe[data-secret="' + data.secret + '"]' );
+		const blockquotes = document.querySelectorAll(
+			'blockquote[data-secret="' + data.secret + '"]'
+		);
+		let i;
+		let source;
+		let height;
+		let sourceURL;
+		let targetURL;
```

Comment:
Let's move these variables to the inner blocks where they are used. That's the ES2015 style.

Reply:
Good idea, 43c15f2!